File: Algorithms Advanced/others/Python/AdjacencyMatrix.py
# 4-node graph adjacency matrix
n = 4
adj_matrix = [[0] * n for _ in range(n)]

# Add edges
adj_matrix[0][1] = 1  # Edge from node 0 to node 1
adj_matrix[1][2] = 1  # Edge from node 1 to node 2

# Display
for row in adj_matrix:
    print(row)

File: Algorithms Advanced/others/C++/AdjacencyMatrix.cpp
#include <iostream>
using namespace std;

int main() 
{
    int vertices = 3;
    int adjMatrix[3][3] = {0};

    // Add edges (example graph)
    adjMatrix[0][1] = 1;
    adjMatrix[0][2] = 1;
    adjMatrix[1][2] = 1;

    // Print adjacency matrix
    for (int i = 0; i < vertices; i++) 
    {
        for (int j = 0; j < vertices; j++)
            cout << adjMatrix[i][j] << " ";
        cout << endl;
    }

    return 0;
}
